### PR TITLE
EXPERIMENTAL FOR DISPLAY ONLY: combine 'Native C types for OSSL_PARAM' with test/params_test.c

### PR DESCRIPTION
**Pull request by levitte**
This combines #8377 with #8396

### CODE REVIEW

File: test/params_api_test.c [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        "D78AF684E71DB0C39CFF4E64FB9DB567132CB9C50CC98009FEB820B26F2DED9B"
+
+static int test_params(void)
+{
+    static size_t sz;
+    static int i;
+    static uint64_t u64 = U64VAL;
+    static double d1 = DOUBLE;
+    static unsigned char buffer[200];
+    size_t buffer_l;
```
Comment (richsalz) [inline test/params_api_test.c]: probably needs to be "static"

Comment (levitte) [reply test/params_api_test.c]: Ah!

**Comment by levitte** [comment]
https://github.com/openssl/openssl/pull/8399#issuecomment-469451888 applies here as well

**Comment by levitte** [comment]
#8396 is now completed, so closing this

**Pull request closed without merging**